src/modality-specific-files/behavioral-experiments.md

```diff
@@ -9,17 +15,20 @@ and a guide for using macros can be found at
 -->
 {{ MACROS___make_filename_template("raw", datatypes=["beh"]) }}
 
-In addition to logs from behavioral experiments performed alongside imaging data
-acquisitions, one can also include data from experiments performed with no neural
-recordings.
-The results of those experiments can be stored in the `beh` directory using the same
-formats for event timing (`_events.tsv`), metadata (`_events.json`),
+In addition to logs from behavioral experiments
+performed alongside imaging data acquisitions,
+one MAY also include data from experiments
+performed with no neural recordings.
+The results of those experiments MAY be stored in the `beh` directory
+using the same formats for event timing (`_events.tsv`),
+metadata (`_events.json`),
 physiological (`_physio.tsv.gz`, `_physio.json`)
 and other continuous recordings (`_stim.tsv.gz`, `_stim.json`)
 as for tasks performed during MRI, electrophysiological or other neural recordings.
-Additionally, events files that do not include the mandatory `onset` and
-`duration` columns can still be included, but should be labeled `_beh.tsv`
-rather than `_events.tsv`.
+Additionally, events files
+that do not include the mandatory `onset` and `duration` columns
+MAY be included,
+but SHOULD be labeled `_beh.tsv` rather than `_events.tsv`.
```

Comment:
Shouldn't that be a MUST?

If a file does not have onset, duration then it is an invalid events file.

Reply:
Yes, let's use a MUST.